src/templates/javascript.js

```diff
@@ -1,28 +1,41 @@
+const { prefixColor } = require('../utils');
+
 const NATIVE_HEADER = `import { PlatformColor } from 'react-native';`;
 
-const generateIos = (colors) =>
+const generateIos = (colors, config) =>
   NATIVE_HEADER +
   '\n\n' +
   colors
     .map(
-      (color) => `export const ${color.name} = PlatformColor('${color.name}');`
+      (color) =>
+        `export const ${color.name} = PlatformColor('${prefixColor(
+          color.name,
+          config
+        )}');`
     )
     .join('\n');
 
-const generateAndroid = (colors) =>
+const generateAndroid = (colors, config) =>
   NATIVE_HEADER +
   '\n\n' +
   colors
     .map(
       (color) =>
-        `export const ${color.name} = PlatformColor('@color/${color.name}');`
+        `export const ${color.name} = PlatformColor('@color/${prefixColor(
+          color.name,
+          config
+        )}');`
     )
     .join('\n');
 
-const generateCss = (colors) =>
+const generateCss = (colors, config) =>
   colors
     .map(
-      (color) => `export const ${color.name} = 'var(--color-${color.name})';`
+      (color) =>
+        `export const ${color.name} = 'var(--color-${prefixColor(
```

Comment:
Maybe only use the `prefix` variable here?
```suggestion
        `export const ${color.name} = 'var(--${prefixColor(
```

Reply:
Agree!